### PR TITLE
Please review the code

### PR DESCRIPTION
**Pull request by DennisXie**
Just leave the comments in this PR.

### CODE REVIEW

File: clients/td_client.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        req.UserID = self._user_id
+        req.AppID = self._app_id
+        req.AuthCode = self._auth_code
+        self._api.ReqAuthenticate(req, 0)
+
+    def OnRspAuthenticate(self, pRspAuthenticateField: tdapi.CThostFtdcRspAuthenticateField,
+                          pRspInfo: tdapi.CThostFtdcRspInfoField, nRequestID: int, bIsLast: bool):
+        """called when authenticate success"""
+        if pRspInfo is not None:
+            logging.info(f"authenticate rsp info, ErrorID: {pRspInfo.ErrorID}, ErrorMsg: {pRspInfo.ErrorMsg}")
```
Comment (Jedore) [inline clients/td_client.py:72]: Log format may need to be unified.

File: clients/td_client.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        req.UserID = self._user_id
+        req.Password = self._password
+        req.UserProductInfo = "openctp"
+        self._api.ReqUserLogin(req, 0)
+
+    def OnRspUserLogin(self, pRspUserLogin: tdapi.CThostFtdcRspUserLoginField, pRspInfo: tdapi.CThostFtdcRspInfoField,
+                       nRequestID: int, bIsLast: bool):
+        """called when login responds"""
+        if pRspInfo is not None:
+            logging.info(f"login rsp info, ErrorID: {pRspInfo.ErrorID}, ErrorMsg: {pRspInfo.ErrorMsg}")
```
Comment (Jedore) [inline clients/td_client.py:93]: Why log this info exclued following successful and failed log ?

File: clients/td_client.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        req.InvestorID = self._user_id
+        self._api.ReqSettlementInfoConfirm(req, 0)
+    
+    def OnRspQrySettlementInfoConfirm(self, pSettlementInfoConfirm: tdapi.CThostFtdcSettlementInfoConfirmField, pRspInfo: tdapi.CThostFtdcRspInfoField, nRequestID: int, bIsLast: bool):
+        logging.info("confirm settlement info success")
+        if pRspInfo is not None:
+            logging.info(f"settlemnt confirm rsp info, ErrorID: {pRspInfo.ErrorID}, ErrorMsg: {pRspInfo.ErrorMsg}")
+    
+    def processConnectResult(self, messageType: str, pRspInfo: tdapi.CThostFtdcRspInfoField, pRspUserLogin: tdapi.CThostFtdcRspUserLoginField = None):
+        data = CTPObjectHelper.object_to_dict(pRspUserLogin) if pRspUserLogin else {}
```
Comment (Jedore) [inline clients/td_client.py:115]: Put `if .. else ...` into `object_to_dict()` seems more elegant ?

File: clients/td_client.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+        req = tdapi.CThostFtdcQryInstrumentField()
+        CTPObjectHelper.dict_to_object(request["QryInstrument"], req)
+        requestId = request["RequestID"]
+        return self._api.ReqQryInstrument(req, requestId)
+    
+    def OnRspQryInstrument(self, pInstrument: tdapi.CThostFtdcInstrumentField, pRspInfo: tdapi.CThostFtdcRspInfoField, nRequestID: int, bIsLast: bool):
+        data = CTPObjectHelper.object_to_dict(pInstrument)
+        rsp = CTPObjectHelper.object_to_dict(pRspInfo)
+        self._rsp_callback({
+            "MessageType": "OnRspQryInstrument",
```
Comment (Jedore) [inline clients/td_client.py:133]: Define `OnRspQryInstrument` as a constraint variable would be better.

File: main.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+from utils import GlobalConfig
+from apps.td_app import app as td_app
+
+app = fastapi.FastAPI()
+app.mount("/td", td_app)
+
+
+async def main():
+    GlobalConfig.load_config("config.json")
+    server_config = uvicorn.Config("main:app", host="0.0.0.0", port=GlobalConfig.Port, log_level="info")
```
Comment (Jedore) [inline main.py:13]: Define `log_level` as a optional config would be more elegant.

File: utils/ctp_object_helper.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+class CTPObjectHelper(object):
+
+    @staticmethod
+    def object_to_dict(obj: object) -> dict[str, any]:
+        data = {}
+        for attr, value in obj.__dict__.items():
+            if attr[0].isupper():
```
Comment (Jedore) [inline utils/ctp_object_helper.py:7]: Should be more careful to use this fictitious rule.

File: clients/td_client.py (hunk 3 above)
Comment (DennisXie) [reply clients/td_client.py:115]: Good idea.

File: utils/ctp_object_helper.py (hunk 6 above)
Comment (DennisXie) [reply utils/ctp_object_helper.py:7]: > Should be more careful to use this fictitious rule.

If we don't make this assumption, we need to write a lot of analogous code. Or maybe we can copy the fields in the obj.__dict__ except the fields are surely no use.

Comment (Jedore) [reply utils/ctp_object_helper.py:7]: Yeah, but it's only a assumption, not certain. Should not B

**Pull request merged**